frontend/packages/console-dynamic-plugin-sdk/src/api/useResolvedExtensions.ts

```diff
@@ -48,38 +40,30 @@ export const useResolvedExtensions = <E extends Extension>(
 
   const [resolvedExtensions, setResolvedExtensions] = React.useState<ResolvedExtension<E>[]>([]);
   const [resolved, setResolved] = React.useState<boolean>(false);
-  const [error, setError] = React.useState<any>(undefined);
+  const [errors, setErrors] = React.useState<any>(undefined);
 
   React.useEffect(() => {
     let disposed = false;
 
-    Promise.all(
+    // eslint-disable-next-line promise/catch-or-return
```

Comment:
@christianvogt FYI the ESLint `promise/catch-or-return` rule is meaningless for `Promise.allSettled` function.

Reply:
Interesting. I wonder if there's an update or a setting for this rule.

Reply:
@vojtechszocs did you look into this?

Reply:
@christianvogt Looking at [`promise/catch-or-return`](https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/catch-or-return.md) rule documentation, there's only `terminationMethod` option which isn't useful in this particular case.

Reply:
Added a comment explaining the suppression of `promise/catch-or-return` rule.